finality-aleph/src/new_network/data_network/aleph_network.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{
```

Comment:
I think both this and RMC should be in a separate module, say `new_network::aleph` since they only use `DataNetwork` and include various AlephBFT related types, while `DataNetwork` is agnostic wrt the data being sent through it.

There might be some iffiness with splitting, if there is feel free to ignore, this is at best a nit anyway.

Reply:
Ignored as you told me to in comment below